src/apps/apploader/app.js

```diff
@@ -140,21 +161,22 @@ define(function(require) {
 		bindEvents: function(parent, appList) {
 			var self = this,
 				updateAppInfo = function updateAppInfo(id) {
-					var app = appList.filter(function(v) { return id === v.id; })[0];
-
-					parent.find('.app-description')
-							.find('h4')
-							.text(app.label)
-							.addBack()
-							.find('p')
-							.text(app.description);
+					var app = _.find(appList, { id: id }),
+						isLink = _.includes(_.keys(monster.config.whitelabel.appLinks), id);
+
+					parent.find('.app-description h4')[isLink ? 'addClass' : 'removeClass']('is-link');
```

Comment:
I think it would be better to save the reference for the element `.app-description` as a variable at `bindEvents`, to reduce the search scope each time an app icon is hovered.

Reply:
Good call, fixed!